.github/buildomat/jobs/package.sh

```diff
@@ -5,10 +5,14 @@
 #: target = "helios-latest"
 #: rust_toolchain = "1.66.1"
 #: output_rules = [
-#:	"=/work/package.tar.gz",
+#:	"=/work/global-zone-packages.tar.gz",
 #:	"=/work/zones/*.tar.gz",
 #: ]
 #:
+#: [[publish]]
```

Comment:
👀

Reply:
See: https://github.com/oxidecomputer/buildomat#configuration , under `publish` specifically

I think this'll be necessary for the `helios` repo to be able to "pick up these output artifacts" by a particular commit of Omicron

Reply:
I'm doing that work in https://github.com/oxidecomputer/helios/pull/56